src/uct/ib/base/ib_device.h

```diff
@@ -349,9 +349,9 @@ int uct_ib_device_is_port_roce(uct_ib_device_t *dev, uint8_t port_num);
 
 
 /**
- * @return 1 if the gid_raw is 0, 0 otherwise.
+ * @return whether the gid is valid
  */
-int uct_ib_device_is_gid_raw_empty(uint8_t *gid_raw);
+int uct_ib_device_is_valid_gid(const union ibv_gid *gid);
```

Comment:
Minor: -> `is_gid_valid`
(consistency w similar place. "is_noun_adjective")

Reply:
renamed it